spec/features/ialmax/saml_sign_in_spec.rb

```diff
@@ -9,7 +9,9 @@
         user = user_with_piv_cac
         visit_idp_from_saml_sp_with_ialmax
         signin_with_piv(user)
+        click_submit_default
```

Comment:
can you explain what these extra submits are for? are they the non-JS SAML form post redirects that usually get submitted by JS?

Reply:
Yeah, that's correct. It's a real pain!

Reply:
We might be able to avoid them by switching to the Chrome driver (JavaScript enabled), which is more realistic to the end-user experience but also slower to run, and may end up with the same amount of overall changes needed (i.e. littering `js: true` throughout all these specs). 🤷

Reply:
I logged a JIRA for that, since it sounds like it's worth thinking more about pros and cons to updating to use the Chrome driver: https://cm-jira.usa.gov/browse/LG-7800